validator/sawtooth_validator/journal/publisher.py

```diff
@@ -35,6 +37,30 @@
 LOGGER = logging.getLogger(__name__)
 
 
+class TransactionSender(object):
```

Comment:
The networking layer has moved in the direction of not providing central services or routing for components. I was following the the pattern of the BlockSender but saw no reason to encumber consensus algorithms with the details on how to build blocks.